lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -210,17 +211,26 @@ def test_no_ccx_created(self):
     def test_create_ccx(self):
         """
         Create CCX. Follow redirect to coach dashboard, confirm we see
-        the coach dashboard for the new CCX.
+        the coach dashboard for the new CCX. Also, confirm that the coach
+        has staff access to it.
         """
+
         self.make_coach()
         url = reverse(
             'create_ccx',
             kwargs={'course_id': self.course.id.to_deprecated_string()})
```

Comment:
@sarina We already do that in a few places when it comes to getting the serialized course ids out.  For example in `student/views.py`, we use that when generating the url for the Choose Your Track page.